common/telemetry/grpc.go

```diff
@@ -141,9 +140,8 @@ func (c *customServerStatsHandler) HandleRPC(ctx context.Context, stat stats.RPC
 			for key, values := range s.Header {
 				span.SetAttributes(attribute.StringSlice("rpc.request.headers."+key, values))
 			}
-			deadline, ok := ctx.Deadline()
-			if ok {
-				span.SetAttributes(attribute.String("rpc.request.timeout", deadline.Format(time.RFC3339Nano)))
+			if deadline, ok := ctx.Deadline(); ok {
+				span.SetAttributes(attribute.Int64("rpc.request.deadline", deadline.UnixNano()))
```

Comment:
I realized "deadline" is clearer.

Reply:
I think both of them make sense. "timeout" should be a duration though. `deadline-now`.